webhook.go

```diff
@@ -35,8 +35,8 @@ type WebhookParams struct {
 	Components      []MessageComponent      `json:"components"`
 	Embeds          []*MessageEmbed         `json:"embeds,omitempty"`
 	AllowedMentions *MessageAllowedMentions `json:"allowed_mentions,omitempty"`
-	// NOTE: Works only for followup messages.
-	Flags uint64 `json:"flags,omitempty"`
+	// NOTE: Works only for followup messages and only MessageFlagsSuppressEmbeds can be set
```

Comment:
Actually `flags` are supported outside of followup endpoints. Additionally, the followup create endpoint exclusively allows for `EPHEMERAL` flag.
So I think it would be better to rephrase this.
```suggestion
	// Only MessageFlagsSuppressEmbeds and MessageFlagsEphemeral can be set.
	// MessageFlagsEphemeral can only be set when using Followup Message Create endpoint.
```